src/main/kotlin/org/rust/ide/surround/DelimiterSurrounder.kt

```diff
@@ -0,0 +1,33 @@
+package org.rust.ide.surround
+
+import com.intellij.lang.surroundWith.Surrounder
+import com.intellij.openapi.editor.Document
+import com.intellij.openapi.editor.Editor
+import com.intellij.openapi.editor.SelectionModel
+import com.intellij.openapi.project.Project
+import com.intellij.openapi.util.TextRange
+import com.intellij.psi.PsiElement
+import com.intellij.psi.PsiWhiteSpace
+import org.rust.lang.core.psi.RustElementFactory
+import org.rust.lang.core.psi.RustExpr
+
+class DelimiterSurrounder(private val left: String,
+                          private val right: String,
+                          private val desc: String) : Surrounder {
```

Comment:
This is better called `description`. In general it is better not to abbreviate names, especially if it is not a name of a local variable. (Names of many Rust Psi classes are abbreviated, but this is to match the original parser and Rust short keywords)